app/konnect/dev-portal/index.md

```diff
@@ -9,5 +9,13 @@ document and manage Application registration for your Services. Admin roles publ
 
 The Dev Portal lives at a separate URL from Konnect and requires that all users, including Admin roles, [register as a Developer](konnect/dev-portal/developers/dev-reg/). 
 
+## Customize the Dev Portal appearance
+
+The {{site.konnect_short_name}} Dev Portal can be customized via the Appearance settings. 
+
+To access the Appearance settings, click **Dev Portal** then **Appearance**.
+
+Here, you will be able to add a logo, select a pre-set theme, and modify background colors, text colors, button colors, and fonts. 
+
 See also:
```

Comment:
```suggestion
## See also
```
Since you added the customization section, this now looks like it's part of the customization section. I suggest making it into its own section; it'll eventually have more than one link, I imagine.

Reply:
@falondarville Regarding sizing for the logo and favicon, the UI states max 1mb for image size, but does the user need more specific info? such as:
Favicon requirements:
Supported image file types: GIF, JPG, PNG, ICO
Maximum size (pixels): 32w x 32h
Must be a square shape
Maximum file size up to 1 MB

Logo requirements:
Supported image file types: GIF, JPG, PNG
Maximum size (pixels): 200w x 41h
Maximum file size up to 1 MB